server/modules/MainServer.js

```diff
@@ -51,8 +51,6 @@ function createExpressApp() {
   app.use(cookieParser());
   app.use(bodyParser.urlencoded({ extended: false, limit: '50mb' }));
   app.use(bodyParser.json({ extended: false, limit: '50mb' }));
-  app.use(cookieAuthentication.middleware);
```

Comment:
Moving the authentication middleware after the logging middleware will mean that we can't log the username associated with a request.